assets/core.styl

```diff
@@ -71,7 +71,9 @@ resets(arr)
   li
     list-style-type: none
     padding-left: LIST_STYLE_OUTER_WIDTH
-    position: relative
+
+    > .ql-ui
+      position: static;
```

Comment:
There's already a similar rule for .ql-ui https://github.com/quilljs/quill/blob/e33f352b9ede51759cb3d616835d1e1e80694251/assets/core.styl#L211-L212

Do we need to add this or just remove the position: relative?

Reply:
The `position: static` is added here to override the `position: absolute` that you linked since we are removing `position: relative` from `li`. Otherwise the checklist icon will be positioned incorrectly:


https://github.com/quilljs/quill/assets/635902/3c68257a-1614-4d45-a385-987295036300

Reply:
Okay so we need it for other ql-ui's (ex code block's language selector) but not list items' - I would move the two rules closer to each other then so it's clear this is intentional:

```diff
diff --git a/assets/core.styl b/assets/core.styl
index 1182f18c..b16de126 100644
--- a/assets/core.styl
+++ b/assets/core.styl
@@ -72,9 +72,6 @@ resets(arr)
     list-style-type: none
     padding-left: LIST_STYLE_OUTER_WIDTH
 
-    > .ql-ui
-      position: static;
-
     > .ql-ui:before
       display: inline-block
       margin-left: -1*LIST_STYLE_OUTER_WIDTH
@@ -213,6 +210,10 @@ resets(arr)
   .ql-ui
     position: absolute
 
+  li
+    > .ql-ui
+      position: static;
+
 .ql-editor.ql-blank::before
   color: rgba(0,0,0,0.6)
   content: attr(data-placeholder)
```